src/components/Lobby/Lobby.jsx

```diff
@@ -34,10 +34,10 @@ export class Lobby extends React.Component {
 
     render() {
         return (
-            <div className={styles.lobbyPage}>
+            <div>
```

Comment:
Maybe put some className in that div, or if it's not gonna have any just wrap the return in <> </>?